platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummary.kt

```diff
@@ -173,20 +181,23 @@ class Summary(
     fun textForConsole(cacheActionText: String, htmlReportFile: File? = null): String {
         val documentationRegistry = DocumentationRegistry()
         return StringBuilder().apply {
-            appendLine()
-            appendSummaryHeader(cacheActionText, totalProblemCount)
-            appendLine()
-            topProblemsForConsole().forEach { problem ->
-                append("- ")
-                append(problem.userCodeLocation.capitalized())
-                append(": ")
-                appendLine(problem.message)
-                problem.documentationSection?.let {
-                    appendLine("  See ${documentationRegistry.getDocumentationFor(it.page, it.anchor)}")
+            // When build degrades gracefully, we keep the console output minimal but still want to see the report link
```

Comment:
:question: The way the link is displayed is controlled by the `ConfigurationCacheProblems.report`. It will skip reporting the link if `totalProblemCount == 0`. It doesn't look like we're asserting for the link presence in the build output.